internal/esmapping/command.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package esmapping
```

Comment:
Let's not treat `internal` as a random dump, keep some organization:
internal/storage/elasticsearch/mapping